.github/CODEOWNERS

```diff
@@ -146,26 +146,26 @@
 
 # Dev workflow team.
 /.circleci/ @U8NWXD @kevintab95
-/.eslintignore @DubeySandeep
-/.eslintrc @DubeySandeep
-/.htmllintrc @DubeySandeep
-/.coveragerc @DubeySandeep
+/.eslintignore @Hudda @DubeySandeep
+/.eslintrc @Hudda @DubeySandeep
+/.htmllintrc @Hudda @DubeySandeep
+/.coveragerc @Hudda @DubeySandeep
 /.gitattributes @DubeySandeep
 /.gitignore @DubeySandeep
-/.isort.cfg @DubeySandeep
-/.pylintrc @DubeySandeep
-/.stylelintrc @DubeySandeep
+/.isort.cfg @Hudda @DubeySandeep
+/.pylintrc @Hudda @DubeySandeep
+/.stylelintrc @Hudda @DubeySandeep
 /.yarnrc @DubeySandeep
 /codecov.yml @nithusha21
-/core/templates/css/.stylelintrc @DubeySandeep
-/tox.ini @DubeySandeep
+/core/templates/css/.stylelintrc @Hudda @DubeySandeep
+/tox.ini @Hudda @DubeySandeep
 /scripts/*.py @DubeySandeep
 /scripts/check_e2e_tests_are_captured_in_ci*.py @kevintab95 @DubeySandeep
 /scripts/check_frontend_test_coverage*.py @DubeySandeep @nithusha21
 /scripts/create_topological_sort_of_all_services*.py @DubeySandeep
 /scripts/install_prerequisites.sh @DubeySandeep
 /scripts/typescript_checks*.py @DubeySandeep @vojtechjelinek
-/scripts/linters/ @DubeySandeep
+/scripts/linters/ @Hudda @DubeySandeep
```

Comment:
Should we assign @sajalasati for the python linter scripts? (And you can own custom eslint checks?)

Reply:
@sajalasati is it fine to assign you to python linters?

Reply:
@DubeySandeep Thanks for checking (I was waiting to first discuss it with you offline). Yes I think I can co-own the following files:
```
codeowner_linter*.py
pylint_extensions*.py
python_linter*.py
```
Should I create a separate PR for this?

Reply:
Let's have one PR to do these simple changes!

@Hudda Can you please update the PR? :)

Reply:
Done. PTAL!

Reply:
LGTM!